samples/Kconfig

```diff
@@ -17,10 +17,11 @@ config NCS_SAMPLES_DEFAULTS
 
 if LOG
 
-# LOG_DEFAULT_LEVEL is declared in Zephyr and also here for a second time,
-# for set the valid default size value
+# LOG_DEFAULT_LEVEL is declared in Zephyr,
+# we declare it here for a second time, to set a different default.
+# Omit the `prompt` to prevent it from appearing twice in two diffent places.
+
```

Comment:
"to set a different default"

If the comment is accurate, does that mean we should omit the help text and range as well?